assets/component-show-more.css

```diff
@@ -0,0 +1,16 @@
+
+.button-show-more {
+  padding-left: 0;
+  justify-content: flex-start;
+  padding-bottom: 1.1rem;
+}
+
+.button-show-more, 
+.button-show-less {
+  margin-top: 1.5rem;
+}
+
+.show-more-wrap ~ show-more-button .label-show-less,
```

Comment:
This CSS seems unnecessarily complex to me, is there a way for the `show-more-button` element to have a class which indicates what state it is in instead?

Reply:
How's this? https://github.com/Shopify/dawn/pull/1443/commits/f1d85ca24677df80bee6e7e0bc5fcffd68516b8d